docs/plans/2026-05-02-hydration-import-blast-radius.md

```
@@ -0,0 +1,234 @@
+}
+```
+
+**Step 2: Run test to verify it fails**
+
+Run: `cargo test --bin quorum caller_blast_radius_ignores_body_only_edits`
+Expected: FAIL — callers will be populated because the current condition matches any overlap.
+
+**Step 3: Write minimal fix**
+
+Change line 76 from:
+```rust
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)